### ISSUE TITLE
Sample weight 

### ISSUE THREAD

**Issue by rebeen** [ZongxianLee/MMD_Loss.Pytorch#1]
I am trying to use this MMD loss function with Keras (I know this is PyTorch ) but I think we cannot use sample weight with this loss function, could you please let me know how to update it to be suitable when we have sample wight 

Thank you
